.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "shellscape",
+  "extends": ["shellscape", "plugin:react/recommended"],
```

Comment:
I could extend the ESLint configuration at the example to limit this change there.

Reply:
@bebraw ^